pkg/netutil/netutil_test.go

```diff
@@ -182,11 +183,13 @@ func TestURLsEqual(t *testing.T) {
 			a:      []url.URL{{Scheme: "http", Host: "example.com:2379"}},
 			b:      []url.URL{{Scheme: "https", Host: "10.0.10.1:2379"}},
 			expect: false,
+			err:    errors.New(`"http://10.0.10.1:2379"(resolved from "http://example.com:2379") != "https://10.0.10.1:2379"(resolved from "https://10.0.10.1:2379")`),
```

Comment:
hmm, `http://10.0.10.1:2379` from `http://example.com:2379` do `==` to `https://10.0.10.1:2379` (resolved from `"https://10.0.10.1:2379"`) ?

I am not sure why this test fails.

Reply:
Scheme is different :)

Reply:
oh, i see now. sorry i didn't notice the scheme change.